javascript/javascript3/week3/shoping_cart.js

```diff
@@ -0,0 +1,134 @@
+console.log('Shoping cart');
+
+                /* Part 1 */
+
+     class Product {
+     constructor(name, price) {
+      this.name = name;
+      this.price = price;
+     }
+
+     async convertToCurrency(curr) {
+        const currency=curr.toUpperCase();
+        let price = 0;
+        const rice = await fetch("https://api.exchangeratesapi.io/latest?base=DKK")
+          .then((promise) => promise.json())
+          .then((data) => {
+            for (let item in data.rates) {
+              if (item == currency) {
+                const currentrate = data.rates[item];
+                price = this.price * currentrate.toFixed(2);
+              }
+            }
+          });
+        return 'Price of '+this.price+' DKK is : '+ price+' '+currency;
+      }
+    }
+  
+  
+     class ShoppingCart {
+     constructor() {
```

Comment:
indentation => are you using [.prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)?

Reply:
installed prettier and formated code with .prettier